02-electric/electric-section-two/electric-section-two.css

```diff
@@ -1,31 +1,79 @@
+.electric-section-two-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-content: center;
+  row-gap: var(--items-gap-mobile);
+  padding: var(--container-padding-mobile);
+}
+.electric-section-two-container > * {
+  border: solid 1px black;
+}
 
-.electric-section-two-container{
-    display: grid;
-    grid-template-columns: 1fr;
-    justify-content: center;
-    row-gap: 15px;
-    padding: 25px
-  
-  }
-  .electric-section-two-container > *{
-      border: solid  1px black;
-  }
-  
-  .electric-section-two-img-container{
-      grid-column: 1/2;
-      
-  }
-  
-  .electric-section-two-title{
-      grid-column: 1/2;
-     text-align: center;
-  }
-  .electric-section-two-content{
-      grid-column: 1/2;
-     text-align: justify;
-  }
-  
-  .electric-section-two-img{
-    display: block;
-      width: 100%;
-  }
\ No newline at end of file
+.electric-section-two-title {
+  text-align: center;
+  font-size: var(--title-size-mobile);
+}
+
+/*******PART ONE*****/
+
+.electric-section-two-part-one-title {
+  text-align: center;
+  font-size: var(--title-size-mobile);
+}
+
+.electric-section-two-part-one-subtitle {
+  text-align: center;
+  font-size: var(--subtitle-size-mobile);
+}
+
+.electric-section-two-part-one-paragraph {
+  grid-column: 1/2;
+  text-align: justify;
+}
+
```

Comment:
add empty selectors for the image containers please (for later)

Reply:
done

Reply:
i willkeep this one like this until the new design which add by karleen then with that i will change this as well